libiberty/d-demangle.c

```diff
@@ -1408,8 +1412,12 @@ dlang_parse_mangle (string *decl, const char *mangled,
 	      string_prepend (decl, " ");
 	      string_prependn (decl, type.b, string_length (&type));
 	    }
+	  if (options & DMGL_VERBOSE)
+	    string_prependn (decl, attr.b, string_length (&attr));
```

Comment:
Wouldn't you want attributes to be postfixed, and not prefixed?

Also, I've just noticed that my addition of `options` should probably strip off `DMGL_TYPES` in the recursive calls.  Otherwise you'll get `int int some.symbol(some.other.symbol)`